src/amber/cli/templates/scaffold/controller/granite/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -63,4 +63,26 @@ class <%= @name.capitalize %>Controller < ApplicationController
     end
     redirect_to "/<%= @name %>s"
   end
+
+  def <%= @name %>_params
+    params.validation do
+      <%- @fields.each do |field| -%>
+      <%- unless %(created_at updated_at).includes? field.name -%>
```

Comment:
This conditional doesn't seem to have a body to execute if true.

Reply:
The intention is not to render the `created_at` and `updated_at` for params validations